e2e/e2e_test.go

```diff
@@ -45,74 +39,12 @@ func setDefaultKubeconfig() {
 	}
 }
 
-// removeCephCSIResource is a temporary fix for CI to remove the ceph-csi resources deployed by rook
-func removeCephCSIResource() {
```

Comment:
(clarification) why is this no longer needed? Rook still rolls out the CSI artifacts, afaict.

Reply:
Hi @ShyamsundarR Thanks for your review :) I recheck the file, and found the code delete following objects.

```
csi-cephfsplugin
csi-rbdplugin
csi-rbdplugin-provisioner
csi-cephfsplugin-provisioner
csi-rbdplugin-provisioner
csi-cephfsplugin-provisioner

rbd provisioner rbac
rbd nodeplugin rbac
cephfs provisioner rbac
cephfs nodeplugin rbac
```

but our e2e code looks like deploy this and delete after e2e.

```golang
func deployRBDPlugin() {
	// deploy provisioner
	framework.RunKubectlOrDie("create", "-f", rbdDirPath+rbdProvisioner)
	framework.RunKubectlOrDie("apply", "-f", rbdDirPath+rbdProvisionerRBAC)
	// deploy nodeplugin
	framework.RunKubectlOrDie("create", "-f", rbdDirPath+rbdNodePlugin)
	framework.RunKubectlOrDie("apply", "-f", rbdDirPath+rbdNodePluginRBAC)
}

func deleteRBDPlugin() {
	_, err := framework.RunKubectl("delete", "-f", rbdDirPath+rbdProvisioner)
	if err != nil {
		e2elog.Logf("failed to delete rbd provisioner %v", err)
	}
	_, err = framework.RunKubectl("delete", "-f", rbdDirPath+rbdProvisionerRBAC)
	if err != nil {
		e2elog.Logf("failed to delete provisioner rbac %v", err)
	}
	_, err = framework.RunKubectl("delete", "-f", rbdDirPath+rbdNodePlugin)
	if err != nil {
		e2elog.Logf("failed to delete nodeplugin %v", err)
	}
	_, err = framework.RunKubectl("delete", "-f", rbdDirPath+rbdNodePluginRBAC)
	if err != nil {
		e2elog.Logf("failed to delete nodeplugin rbac %v", err)
	}
}
```

Reply:
@WoohyungHan all CSI related resources are created in rook-ceph namespace. only thing is you need to delete the cluster role and cluster role binding for both cephfs and rbd created by the rook.

as we are using `kubectl  apply`  while creating RBAC for csi this will be updated with new changes

https://travis-ci.org/ceph/ceph-csi/jobs/620627227#L471-L477
https://travis-ci.org/ceph/ceph-csi/jobs/620627227#L749-L757

I think a separate patch should be good to remove RBAC created by rook and replace `apply`  with `create`

Reply:
I added this as issue #731